helm_charts/stunner_helm/values.yaml

```diff
@@ -0,0 +1,105 @@
+# Default values for stunner_helm.
+
+turnServer:
```

Comment:
Wishlist item: can we change `turnServer` to something less general like `stunner` or `stunner_service` or similar? Our inspiration is to develop STUNner into so much more than a mere TURN server, so maybe it's good to reflect this ambition here. Not insisting though, just an idea